client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -183,21 +211,21 @@ export default React.createClass( {
 
 				<Input label={ this.translate( 'Postal Code', { textOnly } ) } { ...fieldProps( 'postal-code' ) }/>
 
-				<PrivacyProtection
-					cart={ this.props.cart }
-					countriesList= { countriesList }
-					disabled={ formState.isSubmitButtonDisabled( this.state.form ) }
-					fields={ this.state.form }
-					onButtonSelect={ this.handlePrivacyDialogButtonSelect }
-					onDialogClose={ this.handlePrivacyDialogClose }
-					onDialogOpen={ this.handlePrivacyDialogOpen }
-					onDialogSelect={ this.handlePrivacyDialogSelect }
-					isDialogVisible={ this.state.isDialogVisible }
-					productsList={ this.props.productsList } />
+				{ ( abtest( 'privacyCheckbox' ) !== 'checkbox'
+					? this.renderPrivacySection()
+					: this.renderSubmitButton() ) }
 			</div>
 		);
 	},
 
+	handleCheckboxChange() {
+		if ( cartItems.getDomainRegistrationsWithoutPrivacy( this.props.cart ).length > 0 ) {
+			addPrivacyToAllDomains();
+		} else {
+			removePrivacyFromAllDomains();
```

Comment:
Correct, that's also how the current version works. This is under the assumption that if the user wants to protect their contact info, they will want to do so on all domains (and they can remove any from the cart if really needed).